superset-frontend/src/dashboard/components/StickyVerticalBar.tsx

```diff
@@ -0,0 +1,83 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { StickyContainer, Sticky } from 'react-sticky';
+import { styled } from '@superset-ui/core';
+import cx from 'classnames';
+
+export const SUPERSET_HEADER_HEIGHT = 59;
+
+const Wrapper = styled.div`
+  position: relative;
+  width: 16px;
+  flex: 0 0 16px;
+  /* &.animated {
```

Comment:
```suggestion
  /* these animations (which can be enabled with the "animated" class) look glitchy due to chart resizing */
  /* keeping these for posterity, in case we can improve that resizing performance */
  /* &.animated {
```